templates/cis.rules.erb

```diff
@@ -107,7 +107,11 @@
 -a always,exit -F path=/sbin/unix_chkpwd -F perm=x -F auid>=1000 -F auid!=4294967295 -k privileged
 -a always,exit -F path=/sbin/netreport -F perm=x -F auid>=1000 -F auid!=4294967295 -k privileged
 -a always,exit -F path=/sbin/pam_timestamp_check -F perm=x -F auid>=1000 -F auid!=4294967295 -k privileged
--a always,exit -F path=/lib64/dbus-1/dbus-daemon-launch-helper -F perm=x -F auid>=1000 -F auid!=4294967295 -k privileged
+<% dbus = [
```

Comment:
@bdwyertech. Thanks for raising a PR. I just need a bit more information to understand the need for this change.
What version of RHEL is this issue present on?
Currently, the rules specified in this file match the rules which CIS specify in their benchmark, meaning a change to this rule would result in failing CIS Inspec tests.